internal/databases/postgres/wal_push_handler.go

```diff
@@ -32,21 +31,20 @@ func (err CantOverwriteWalFileError) Error() string {
 
 // TODO : unit tests
 // HandleWALPush is invoked to perform wal-g wal-push
-func HandleWALPush(uploader *WalUploader, walFilePath string) {
```

Comment:
You should also modify the original ```wal-push``` command handler (cmd/pg/wal_push.go):
1. Check for the returned error
2. Since the uploading folder initialization is removed from ```HandleWALPush```, add it before the function call.